src/dz27/oracle

```diff
@@ -0,0 +1,28 @@
+1
+SELECT * FROM classicmodels.employees
+left join customers on customers.salesRepEmployeeNumber=employees.employeeNumber
+where customers.country='Paris' or customers.country="Tokio";
```

Comment:
Your task was `Show employees working in Paris and Tokyo`, It's join between employees and customers